bot/seasons/evergreen/movie.py

```diff
@@ -0,0 +1,211 @@
+        await ImagePaginator.paginate(pages, ctx, embed)
+
+    async def get_movies_list(self, client: ClientSession, genre_id: str, page: int) -> Dict:
+        """Return JSON of TMDB discover request."""
+        # Define params of request
+        params = {
+            "api_key": TMDB_API_KEY,
+            "language": "en-US",
+            "sort_by": "popularity.desc",
+            "include_adult": "false",
+            "include_video": "false",
+            "page": page,
+            "with_genres": genre_id
+        }
+
+        # Create URL
+        url = BASE_URL + "discover/movie?" + urlencode(params)
+
+        # Make discover request to TMDB
+        async with client.get(url) as resp:
+            data = await resp.json()
+
+        # Return response result
+        return data
+
+    async def get_pages(self, client: ClientSession, movies: Dict, amount: int) -> (List[Tuple[str, str]]):
+        """Fetch all movie pages from movies dictionary. Return list of pages."""
+        pages = []
+
+        for i in range(amount):
+            # Get movie ID, fetch movie information
+            movie_id = movies['results'][i]['id']
+            movie = await self.get_movie(client, movie_id)
+
+            # Create page, append it to pages
+            page, img = await self.create_page(movie)
+            pages.append((page, img))
+
+        return pages
+
+    async def get_movie(self, client: ClientSession, movie: int) -> Dict:
+        """Get Movie by movie ID from TMDB. Return result dictionary."""
+        # Define URL params, generate URL
+        params = {
+            "api_key": TMDB_API_KEY,
+            "language": "en-US"
+        }
+        url = BASE_URL + f"movie/{movie}?" + urlencode(params)
+
+        # Do request, return result
+        async with client.get(url) as resp:
+            return await resp.json()
+
+    async def create_page(self, movie: Dict) -> (str, str):
+        """Create page from TMDB movie request result. Return formatted page + image."""
+        text = ""
+
+        # Add title + tagline (if not empty)
+        text += f"**{movie['title']}**\n"
+        if movie['tagline'] != "":
+            text += f"{movie['tagline']}\n\n"
+        else:
+            text += "\n"
+
+        # Add other information
+        text += f"**Rating:** {movie['vote_average']}/10 :star:\n"
+        text += f"**Release Date:** {movie['release_date']}\n\n"
+
+        text += "__**Production Information**__\n"
+
+        companies = movie['production_companies']
+        countries = movie['production_countries']
+
+        text += f"**Made by:** {', '.join(company['name'] for company in companies)}\n"
+        text += f"**Made in:** {', '.join(country['name'] for country in countries)}\n\n"
+
+        text += "__**Some Numbers**__\n"
+
+        text += f"**Budget:** ${movie['budget'] if movie['budget'] != 0 else '?'}\n"
+        text += f"**Revenue:** ${movie['revenue'] if movie['revenue'] != 0 else '?'}\n"
```

Comment:
Can just use `if movie[string]` instead of checking against zero, as non zero values are truthy.